### DISCUSSION TITLE
Support for k8s 1.22

### DISCUSSION THREAD

**Discussion by khongten1452** [argoproj/argo-rollouts#1688] [Q&A]
Does the lattest release 1.1.1 support k8s 1.22? 

I saw the merge https://github.com/argoproj/argo-rollouts/pull/1529 in the lattest release, but there's no docs in this https://argoproj.github.io/argo-rollouts/installation/

Which flag should I use, and where to put it?  https://github.com/argoproj/argo-rollouts/issues/1507
It's gonna be like the following, isn't it? 
kubectl apply -n argo-rollouts -f https://github.com/argoproj/argo-rollouts/releases/latest/download/install.yaml --ingress-api-version networking.k8s.io/v1
Thanks!

**Comment by harikrongali** [comment]
1529 will be available with 1.2 release which will be end of Jan 2022

**Reply by Wojciech-Chilinski** [reply]
Any update here? How much of delay we might expect?
